client/src/components/Search/SearchBox.js

```diff
@@ -191,7 +196,10 @@ class SearchBox extends Component {
    */
   renderInput() {
     const { id, searchText, onChange, placeholder, name, borders } = this.props;
-    const style = { paddingLeft: `${this.calculateInputLeftPadding()}px` };
+    const style = {
+      paddingLeft: `${this.calculateInputLeftPadding()}px`,
+      paddingRight: `${this.calculateInputRightPadding()}px`
+    };
```

Comment:
Why both left and right, but before it was only doing left padding?

Reply:
Right padding was added to fix the bug :-)

Reply:
From memory it was because there's an "x" that shows when you type in the text input in chrome, and without the right padding change it was totally out of alignment after the search button text was removed